src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  display: flex;
+  margin-bottom: 24px;
+  justify-content: space-between;
+  align-items: center;
+
+  &__star-1 {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__star {
+    border-radius: 0.5px;
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:nth-child(-n + 4) {
```

Comment:
Use the stars block from the previous task. The amount of stars should depend on modifier